mmv1/products/cloudids/terraform.yaml

```diff
@@ -27,6 +27,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       location: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
+      threat_exceptions: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
```

Comment:
What is the reason to set  `ignore_read: true? I think `ignore_read: true` is not needed here and can be removed.

Reply:
Yeah I think it can be removed after change the field name to threatExceptions